tools/build.sh

```diff
@@ -6,15 +6,23 @@
 IMPLEMENTATION=$1 # 'aws' or 'gcp'
 JAVA_SOURCE_ROOT=$2
 
+while getopts ":q" opt; do
+  case $opt in
+    q)
+      QUIET_OPTIONS="-q -Dmaven.test.skip=true"
+      ;;
+  esac
+done
+
+QUIET_OPTIONS="-q -Dmaven.test.skip=true"
```

Comment:
:yellow_circle:  As now quiet is optional, this line it seems wrong -it will enforce it, no matter the setting you provide-. Maybe you want to just initialize it before the while loop to an empty value.